cli/src/main/resources/help/command.txt

```diff
@@ -46,7 +46,13 @@ DESCRIPTION
 
   data-source --jndi-name=my-new-ds --min-pool-size=11 --max-pool-size=22
 
-  Tab-completion will help completing operation and property names.
+  
+ARGUMENTS
+  
+  Arguments should be passed with the commands, in CLI mode command we are having arguments(--help, --help --commands and etc)
+
+  For example, command /subsystem=datasources/data-source.
```

Comment:
Same, do we really need this sentence? The command action (add/remove/list) is missing. Perhaps example should be moved in the "Managing Commands"